teamworkApp/test/models/assignment_test.rb

```diff
@@ -1,7 +1,7 @@
 require 'test_helper'
 
 class AssignmentTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+   test "the truth" do
+     assert true
+   end
```

Comment:
What does this one do?

Reply:
it tests to make sure that true is true (basically that you don't have an error in the fixtures which we did :*( )